packages/truffle/scripts/postinstall.js

```diff
@@ -0,0 +1,20 @@
+const { statSync } = require("fs");
+const { execSync } = require("child_process");
+
+const bundledCLI = "./build/cli.bundled.js";
+
+const postinstallObtain = () => {
+  try {
+    statSync(bundledCLI);
+    execSync(`node ${bundledCLI} obtain --solc=0.5.0`);
+  } catch ({ message }) {
+    if (message.includes("no such file")) return;
+    throw new Error(message);
```

Comment:
We should make sure that this error case looks nice on both `npm` and `yarn` (like what if the obtain fails?)